chaos_kitten/cli.py

```
@@ -175,371 +174,84 @@ def scan(
+
+        # Generate report
+        reporter_cfg = cfg.get("reporting", {})
+        reporter = Reporter(
+            output_path=reporter_cfg.get("output_path", "./reports"),
+            output_format=reporter_cfg.get("format", "html"),
+        )
+        report_file = reporter.generate({"vulnerabilities": vulnerabilities}, target_url)
+        console.print(f"\n📄 Report: [underline]{report_file}[/underline]")
+        
+        # Enforce fail-on-critical
         if fail_on_critical:
```

Ruff (0.15.1)

[error] Undefined name `fail_on_critical`

(F821)